Lib/fontTools/ttLib/tables/_h_h_e_a.py

```diff
@@ -65,6 +65,8 @@ def recalc(self, ttFont):
 			topDict = ttFont['CFF '].cff.topDictIndex[0]
 			for name in ttFont.getGlyphOrder():
 				cs = topDict.CharStrings[name]
+				if not hasattr(cs, 'bounds'):
```

Comment:
This caching of bounds means that modifying a charstring and saving again doesn't get the bounds right.  I would go without caching.  WDYT?  I know it will be twice the work at least... I don't have a better solution right now.